### PR TITLE
fix(usm): simplified sharing tooltip at high zoom

### PR DESCRIPTION
**Pull request by jpan-box**
Make simplified sharing visible at high zoom
by moving from middle-left to bottom-center

Should not merge until we are confident tab order may flow between the tooltip and the primary modal.

QUESTION: The updated snapshots reflect the intended change, HOWEVER, I am wondering why the Tooltip component still shows up in some of these snapshots where I expect the tooltip to NOT render at all, i.e. when isShown prop is NOT true?

Screenshot 2023-01-19 at 4 11 59 PM

Screenshot 2023-01-19 at 4 12 10 PM

### CODE REVIEW

File: src/features/unified-share-modal/UnifiedShareForm.js [hunk 1]
```diff
@@ -482,7 +482,7 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
             className: 'usm-ftux-tooltip',
             // don't want ftux tooltip to show if the recommended sharing tooltip callout is showing
             isShown: !recommendedSharingTooltipCalloutName && shouldRenderFTUXTooltip && showCalloutForUser,
-            position: 'middle-left',
+            position: 'bottom-center',
```
Comment (tjuanitas) [inline src/features/unified-share-modal/UnifiedShareForm.js:485]: it's possible that with a long tooltip text, the problem is now moved to the bottom of the modal i.e. the tooltip becomes off-screen at the bottom.

might not be reproducible with english but could be with other languages.

Comment (tjuanitas) [reply src/features/unified-share-modal/UnifiedShareForm.js:485]: but we don't really have a better solution at the moment besides removing it altogether

File: src/features/unified-share-modal/__tests__/UnifiedShareForm.test.js [hunk 2]
```diff
@@ -106,6 +106,7 @@ describe('features/unified-share-modal/UnifiedShareForm', () => {
                 shouldRenderFTUXTooltip: true,
                 showCalloutForUser: true,
             });
+            expect(wrapper.find('.usm-ftux-tooltip').prop('position')).toBe('bottom-center');
```
Comment (tjuanitas) [inline src/features/unified-share-modal/__tests__/UnifiedShareForm.test.js:109]: not sure the best practice on this but I've generally seen us use the component name unless we need the extra specificity of the class

**Review by tjuanitas — APPROVED** [review]
lgtm

**Comment by jpan-box** [comment]
Closed / Won't Fix

**Pull request closed without merging**